Docker/startozwdaemon.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+shopt -s nocaseglob
+if [ ! -c "$USB_PATH" ]
+then
+	echo "USB Path \"$USB_PATH\" does not exist or is not a Character Device"
+	exit 128
+else
+	CMD="-s $USB_PATH"
+fi
+
+if [ ! -z "$MQTT_SERVER" ]
+then
+	CMD+=" --mqtt-server $MQTT_SERVER"
+	if [ ! -z "$MQTT_PORT" ]
```

Comment:
Wouldn't it make sense to be able to change `MQTT_PORT` without changing `MQTT_SERVER`?

Reply:
MQTT_SERVER defaults to localhost anyway... so you would have to change the server regardless.